Dart/testSrc/com/jetbrains/dart/analysisServer/DartServerCompletionTest.java

```diff
@@ -176,4 +180,18 @@ public void testNoCompletionAfterDigit() {
     myFixture.completeBasic();
     assertNull(myFixture.getLookup());
   }
+
+  public void testNotYetImportedClass() throws InterruptedException {
+    // Wait until we've received symbols to cache from the dart:io library from analysis server.
+    while (true) {
```

Comment:
I removed `while(true)` and added `doHighlighting()` instead. While highlighting goes, available suggestions arrive. 
I removed unused `DartAnalysisServerService..getAvailableSuggestionSets()` and `DartServerData.getAvailableSuggestionSets()`.

Reply:
Without these bits, how can the test know that the symbols from `dart:io` have been cached? If they haven't been sent in an available suggestions notification when we make the completions request, it won't show up in the completion suggestion response.

Reply:
Well, the test doesn't know it for sure. It's possible that the test will fail because of a race. For now, it waits for `DartAnalysisServerService.waitForAnalysisToComplete_TESTS_ONLY()` to return and this time is enough for getting completion data from the server. At least in my environment :). If it fails too often we'll think about a more reliable solution.